packages/contracts/src/dollar/ERC1155Ubiquity.sol

```diff
@@ -24,15 +24,15 @@ contract ERC1155Ubiquity is ERC1155, ERC1155Burnable, ERC1155Pausable {
     // ----------- Modifiers -----------
     modifier onlyMinter() {
         require(
-            manager.hasRole(manager.GOVERNANCE_TOKEN_MINTER_ROLE(), msg.sender),
+            manager.hasRole(manager.UBQ_MINTER_ROLE(), msg.sender),
```

Comment:
Looks like this is a partial rename from our global name changes PR. `GOVERNANCE_TOKEN_MINTER_ROLE` appears to be updated according to the new naming convention. Which is correct?

Reply:
Changed them all to `GOVERNANCE_TOKEN_MINTER_ROLE`